Userland/Libraries/LibJS/Runtime/DateConstructor.cpp

```diff
@@ -156,7 +156,7 @@ void DateConstructor::initialize(GlobalObject& global_object)
     u8 attr = Attribute::Writable | Attribute::Configurable;
     define_native_function(vm.names.now, now, 0, attr);
     define_native_function(vm.names.parse, parse, 1, attr);
-    define_native_function(vm.names.UTC, utc, 1, attr);
+    define_native_function(vm.names.UTC, utc, 7, attr);
```

Comment:
Can we add a test for this? For example:
```js
    test("length is 7", () => {
        expect(Date.UTC).toHaveLength(7);
    });
```

Reply:
Sure, done